keras_cv/layers/preprocessing/random_crop_and_resize_test.py

```diff
@@ -290,3 +292,18 @@ def test_augment_boxes_ragged(self):
         self.assertAllClose(
             expected_output["classes"], output["bounding_boxes"]["classes"]
         )
+
+    def random_crop_and_resize_on_batched_image(self):
+        image_shape = (100, 100, 100, 3)
+        image = tf.random.uniform(shape=image_shape)
+
+        layer = preprocessing_utils.RandomCropAndResize(
+            target_size=self.target_size,
+            aspect_ratio_factor=(3 / 4, 4 / 3),
+            crop_area_factor=(0.8, 1.0),
+        )
+
+        input_resized = tf.image.resize(image, self.target_size)
+        output = layer(image, training=False)
```

Comment:
can we invoke this with training=True as well and check that all of the outputs are different?

Reply:
Sure